coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2726,4 +2726,10 @@ def tensor(context, node):
     # inputs[3] is a bool (not sure what it is)
     shape = mb.shape(x=inputs[2], name=node.name+"_shape")
     context.add(mb.fill(shape=shape, value=val, name=node.name))
+def constant_pad_nd(context, node):
```

Comment:
With the latest sync , this op is already in the master branch now 
(https://github.com/apple/coremltools/blob/6245e289187c95d32b6ff65bc6348310ebbdf71a/coremltools/converters/mil/frontend/torch/ops.py#L912)